src/screens/SSICredentialDetailsScreen/index.tsx

```diff
@@ -26,6 +26,7 @@ enum CredentialTabRoutesEnum {
 }
 
 const SSICredentialDetailsScreen: FC<Props> = (props: Props): JSX.Element => {
+  console.log('Details screen opened.');
```

Comment:
i see a lot of console logs. remove those. if you want to be able to see some extra details when debugging, use debug. you can find examples within the app

Reply:
attempted a fix.

Reply:
okay, I did not understand the 'debug' is a package that we use. started using it now